src/components/Navbar/Navbar.styled.ts

```diff
@@ -91,16 +94,10 @@ const MenuIconBar = styled("span")((props: { isOpen: boolean }) => {
   }
 
   return {
-    transition: `${theme.transitions.create(
-      ["top", "left", "width", "transform"],
-      {
-        duration: theme.transitions.duration.shorter,
-        easing: theme.transitions.easing.easeInOut,
-      }
-    )}`,
+    transition: `all 0.3s ease`,
     height: "2px",
     width: "100%",
-    backgroundColor: theme.palette.text.primary,
+    backgroundColor: "text.primary",
```

Comment:
Is not it necessary to use the theme here?

Reply:
good catch